ar_exercise/exercise.rb

```diff
@@ -34,38 +35,134 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.first
+
 # get all tenants older than 65
+older_tenants = Tenant.where('age > 65')
```

Comment:
Take a look at this post for how to use a conditional in a `.where` statement: http://stackoverflow.com/questions/11317662/rails-using-greater-than-less-than-with-a-where-statement